README.md

```diff
@@ -166,6 +167,62 @@ mysql::db { 'mydb':
 }
 ```
 
+### Install Percona server on Centos
+
+This example shows how to do a minimal installation of a Percona server on a Centos (has been tested on Puppet 4.4 / Centos 7 / Percona Server 5.7) including the Percona server, client and bindings.
+
+The configuration change here is really minimal.
+
+**Note:** The installation of the yum repository is not part of this package and is here only to show a full example of how you can install this.
+
+```puppet
+yumrepo { 'percona':
+  descr    => 'CentOS $releasever - Percona',
+  baseurl  => 'http://repo.percona.com/centos/$releasever/os/$basearch/',
+  gpgkey   => 'http://www.percona.com/downloads/percona-release/RPM-GPG-KEY-percona',
+  enabled  => 1,
+  gpgcheck => 1,
+}
+
+class {'mysql::server':
+  package_name     => 'Percona-Server-server-57',
+  service_name     => 'mysqld',
+  config_file      => '/etc/my.cnf',
+  includedir       => '/etc/my.cnf.d',
+  override_options => {
+    mysqld => {
+      log-error => '/var/log/mysqld.log',
+      pid-file  => '/var/run/mysqld/mysqld.pid',
+    },
+    mysqld_safe => {
+      log-error => '/var/log/mysqld.log',
+    },
+  }
+}
+
+# Note: this is not needed  in our case as installing Percona-Server-server-57
+# also installs Percona-Server-client-57. It is only here to show how to
+# install the Percona MySQL client
+class {'mysql::client':
+  package_name => 'Percona-Server-client-57',
+}
+
+class { 'mysql::bindings':
+  client_dev_package_name => 'Percona-Server-devel-57',
+  daemon_dev_package_name => 'Percona-Server-shared-57',
+}
```

Comment:
The names `client_dev_package_name` and `daemon_dev_package_name` seem confusing. Where do they come from? What does `daemon_dev` mean? I don't see `daemon` in the name of any of the packages.

The Percona MySQL packages are

```
Percona-Server-client-57.x86_64        5.7.11-4.1.el6            percona-release
Percona-Server-devel-57.x86_64         5.7.11-4.1.el6            percona-release
Percona-Server-server-57.x86_64        5.7.11-4.1.el6            percona-release
Percona-Server-shared-57.x86_64        5.7.11-4.1.el6            percona-release
Percona-Server-57-debuginfo.x86_64     5.7.11-4.1.el6            percona-release
Percona-Server-test-57.x86_64          5.7.11-4.1.el6            percona-release
Percona-Server-tokudb-57.x86_64        5.7.11-4.1.el6            percona-release
```

The Oracle MySQL packages are 

```
mysql-community-client.x86_64       5.7.12-1.el6     mysql57-community          
mysql-community-common.x86_64       5.7.12-1.el6     mysql57-community          
mysql-community-devel.x86_64        5.7.12-1.el6     mysql57-community          
mysql-community-embedded.x86_64     5.7.12-1.el6     mysql57-community          
mysql-community-libs.x86_64         5.7.12-1.el6     mysql57-community          
mysql-community-libs-compat.x86_64  5.7.12-1.el6     mysql57-community          
mysql-community-server.x86_64       5.7.12-1.el6     mysql57-community          
mysql-community-test.x86_64         5.7.12-1.el6     mysql57-community          
```